node/node.go

```diff
@@ -154,6 +154,7 @@ func (n *Node) Stop() {
 }
 
 // these methods are using by GUI.
+
```

Comment:
I added a new line not because the comment is related to the`ConsManager ` method; in fact, I included it to indicate to others that these methods are for the GUI, as recommended by the Golang Linter.